mobile/src/board/hooks.tsx

```diff
@@ -0,0 +1,30 @@
+import { useContext, useEffect, useState } from "react";
```

Comment:
call this file `useBoard` as it only exports that hook. 

In general, we export one hook/component per file.